scripts/app.js

```diff
@@ -342,21 +344,21 @@ $(document).ready( function() {
 		$( this ).toggleClass ( 'active' );
 	});
 
-	$( '.overlay' ).click( function( event ) {
+	body.on('click', '.overlay', function( event ) {
 		$('.filters').removeClass('filters-open');
 		$('.nav-filters'). removeClass('active');
 		$('.overlay').fadeToggle(200, 'linear');
 	});
 
-	$('.refreshListings').click(function() {
+	body.on('click', '.refreshListings', function() {
 		shack.resetListings();
 	});
 
 	/**
 	 * highlights active search form filters on click
 	 * @param event - original click event
 	 */
-	$( '.filters__filter-option' ).click( function( event ) {
+	body.on('click', '.filters__filter-option', function( event ) {
```

Comment:
This element isn't dynamically added to the DOM so it can be targeted directly by it's selector as it was